src/Drawings/Areas.js

```diff
@@ -60,7 +60,7 @@ class Areas {
     }
 
     monstersToString(monsters, lang = "en") {
-        let str = "`\n";
+        let str = "`";
```

Comment:
Did you tried? I don't think removing the \n is a good idea.

Reply:
Before
![image](https://user-images.githubusercontent.com/25572454/77260684-58c62280-6c68-11ea-81ec-edecfb7884aa.png)
After
![image](https://user-images.githubusercontent.com/25572454/77260685-5e236d00-6c68-11ea-95d7-ef66c6cd8e86.png)
I guess I forgot to also fix Plants
But removing the \n did exactly what I expected